code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -969,7 +969,7 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	else if(istype(C, /obj/item/card/id))
 		var/obj/item/card/id/idcard = C
-		if(!idcard.registered_name)
+		if(!idcard.registered_name || istype(idcard, /obj/item/card/id/syndicate/broken)) // also rejects a broken chameleon
```

Comment:
I don't like a specific check here, perhaps a variable on the ID itself indicating if it can be inserted into a PDA or not would be better. That way other IDs like clockcult ones or whatever could be rejected too.

Reply:
I will do that when a paperslip ID PR is merged. put this delayed for a while.